src/modm/ui/button.md

```diff
@@ -0,0 +1,64 @@
+# Debouncing Buttons
+
+The `modm::ButtonGroup` class is able to debounce multiple buttons at the same time.
+The buttons have to be low-active. If this isn't the case invert their signal before passing it to the `update()` method.
+
+The `update()` method needs to be called periodically for example every 10ms. Preferred in a timer interrupt function.
+
+The detection for long or repeated presses works only correctly for one key at a time. This constraint only applies to buttons listed in the `mask` variable.
```

Comment:
Can you limit the lines to 100 chars, that way it renders ok on `lbuild discover -n ::button` in the terminal.
It's not reflowed automatically since that messed with some of the formatting.
For the website it doesn't matter either way.